.github/workflows/build.yml

```diff
@@ -53,8 +53,13 @@ jobs:
           path: flutter-tizen
       - name: Build examples of changed packages
         if: ${{ env.HAS_CHANGED_PACKAGES == 'true' }}
+        # Example app of geolocator and permission_handler are temporary excluded 
+        # due to unmigrated depedency "baseflow_plugin_template". 
+        # Reallow the builds by removing the "--exclude=geolocator,permission_handler" line
+        # after https://github.com/Baseflow/baseflow_plugin_template/pull/4 is resolved.
```

Comment:
@HakkyuKim 
It seems that `baseflow_plugin_template_v2.1.1` can be used. 
Can I apply it?

Reply:
@bbrto21 
Thanks! I'll actually update it together with https://github.com/flutter-tizen/plugins/pull/436 as it's a simple script change.

Reply:
Ok.

Reply:
@HakkyuKim I think it should be a separate PR. https://github.com/flutter-tizen/plugins/pull/436 contains too many changes unrelated to its title.

Reply:
OK.